apps/i18n/tutorialExplorer/en_us.json

```diff
@@ -49,5 +49,6 @@
   "tutorialDetailsShortLink": "Short link",
   "tutorialDetailInternationalLanguages": "Languages",
   "headingTutorialsYourLanguage": "Tutorials in your language",
-  "headingTutorialsManyLanguages": "Tutorials in many languages"
+  "headingTutorialsManyLanguages": "Tutorials in many languages",
+  "tutorialDetailStandards": "Standards"
```

Comment:
good point.  talked with @tanyaparker, currently this is grouped by how it appears in the UI, and we think that works okay for now.